snp-dists/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM ubuntu:xenial
+ 
+# metadata
+LABEL base.image="ubuntu:snp-dists"
+LABEL version="1"
+LABEL software="snp-dists"
+LABEL software.version="1.0"
+LABEL description="Convert a FASTA alignment to SNP distance matrix"
+LABEL website="https://github.com/tseemann/snp-dists"
+
+MAINTAINER Kevin Libuit <kevin.libuit@dgs.virginia.gov>
+
+RUN apt-get update && apt-get install -y python \
+  git make libz-dev gcc
+
+RUN cd / &&\
+  git clone https://github.com/tseemann/snp-dists.git && \
```

Comment:
Could you change this line to pull the latest release with `wget` instead of doing `git clone`. Will probably have to add `apt-get install wget` too. Sorry to be picky 🤓

Reply:
Good call. changed to wget to grab latest version (0.6.2); also added the /data workdir